mscore/pagesettings.h

```diff
@@ -22,29 +22,42 @@
 #define __PAGESETTINGS_H__
 
 #include "ui_pagesettings.h"
+#include "libmscore/mscore.h"
+#include "libmscore/page.h"
 
 namespace Ms {
 
 class MasterScore;
 class Score;
 class Navigator;
 
+// For converting from Ms::Units to a two-char string used as a suffix
+static const char* unitSuffixes[] = {"px", "mm", "in", "sp"};
```

Comment:
OK.  What about the literals I pointed out elsewhere in inspectorimage.cpp?  Are those exposed to the user and they should be translated too?  See here:
https://github.com/musescore/MuseScore/blob/master/mscore/inspector/inspectorImage.cpp#L180-L212